vector/src/main/java/im/vector/riotx/features/notifications/NotificationDrawerManager.kt

```diff
@@ -343,12 +343,19 @@ class NotificationDrawerManager @Inject constructor(private val context: Context
                         globalLastMessageTimestamp = lastMessageTimestamp
                     }
 
+                    val tickerText = if (roomEventGroupInfo.isDirect) {
+                        String.format("%s: %s", events[0].senderName, events[0].description)
```

Comment:
We should use string resource for that.
What about defining 2 strings like "%1$s says: %2$s" for the first case and "In room %1$s, %2$s says: %3$s"?

Reply:
As a screen reader user, I dislike when chatroom messages read out as something like "Nolan says: Hey. or "In Matrix HQ: Nolan says What's up!" It only adds a few syllables, but over time slows down the experience. Further, I can usually figure out what's happening after the first message or two, after which point the extra syllables waste time. Riot Web did this for a bit in places, and I recall many of us requested succinctness.

Reply:
ok, I get the point. Anyway we need to use string resources to handle RTL correctly. You get use the same mechanism than https://github.com/vector-im/riotX-android/blob/5081361c2d17fa3a46b3f455b65a5c25dff58641/vector/src/main/res/values/strings.xml#L1310.